src/javascript/app_2/Modules/SmartChart/Containers/smart-chart.jsx

```diff
@@ -41,7 +41,7 @@ class Chart extends React.Component {
             <SmartChart
                 barriers={this.props.barriers_array}
                 bottomWidgets={this.props.should_show_last_digit_stats ? undefined : this.bottomWidgets}
-                chartControlsWidgets={this.chartControlsWidgets}
+                chartControlsWidgets={this.props.is_contract_mode ? null : this.chartControlsWidgets}
```

Comment:
Added this here but it wouldn't work until SmartCharts fixes the issue where it's overriding `null` passed to the `chartControlsWidgets` prop and still showing the chart controls.